image.yaml

```diff
@@ -55,24 +55,48 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: 0.35.1
+                  #url: /tmp/bb/repo/cct_module
+                  ref: 0.39.0
           - name: jboss-eap-modules
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: CD17_CR2
-          - name: sso73_modules
+                  # url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  # ref: 7.3.x-legacy
+                  url: https://github.com/luck3y/jboss-eap-modules.git
+                  #url: /tmp/bb/repo/jboss-eap-modules
```

Comment:
This one too (just local path the `jboss-eap-modules` were copied / tested against).